logs_ingest/main.py

```diff
@@ -127,6 +127,11 @@ def parse_record(record: Dict, self_monitoring: SelfMonitoring):
     category = record.get("category", "").lower()
     infer_monitored_entity_id(category, parsed_record)
 
+    attribute_value_length_limit = get_int_environment_value("DYNATRACE_LOG_INGEST_ATTRIBUTE_VALUE_MAX_LENGTH", 250)
```

Comment:
We could move this statement to line 34, to avoid fetching the value from environment on every record.

Reply:
similar issue in line 136